uber/configspec.ini

```diff
@@ -958,10 +952,12 @@ __many__ = integer
 __many__ = integer
 
 [[stocks]]
-# Use this to set limits on how many copies of each badge can be issues, e.g.
+# Use this to set limits on how many copies of each badge can be sold, e.g.
 # if you only have 200 Friday badges you'd say "friday = 200".  Although you
 # can list arbitrary things here, the intention is for options in this section
-# to correspond to badge types, e.g. "attendee_badge = 2000", etc.
+# to correspond to badge types.
+# We shut down ALL badge sales if attendee badges run out, so this is the event cap.
+attendee_badge = integer(default=20000)
```

Comment:
FWIW, I believe that ``23500`` is the current attendee badge cap.  Though since I imagine we'll put that in the pillar data, it probably doesn't matter what the default is here.